Frontend/ui-driver/src/Screens/ReferralScreen/Controller.purs

```diff
@@ -143,13 +145,15 @@ data ScreenOutput = GoToHomeScreen
 eval :: Action -> ReferralScreenState -> Eval Action ScreenOutput ReferralScreenState
 
 eval (UpdateLeaderBoard (LeaderBoardRes leaderBoardRes)) state = do
+  _ <- pure $ firebaseLogEvent "ny_driver_leaderboard"
```

Comment:
@Anuragini-2506 What is use-case of adding firebase event?

Reply:
@kavya-shree-s pranav asked this to add so that we can track the number of time API is getting hit